src/content/docs/dropins/user-auth/recaptcha.mdx

```diff
@@ -0,0 +1,177 @@
+
+<Tasks>
+
+## Integrations
+
+To integrate ReCaptcha, follow these steps:
+
+<Steps>
+
+1. Load the app.
+
+   - Call the `setConfig` method to fetch and set reCAPTCHA configuration.
+   - (Optional) Extend the configuration with custom forms if a custom integration not natively supported by Adobe Commerce is planned.
+
+1. Load the page with a protected form.
+
+   Call the `initReCaptcha` method to add the script and initialize the ReCaptcha badge.
+
+1. Render the form.
+
+   - The user interacts with the page and form, providing the required data.
+   - The user clicks the submit button.
+
+1. Call the `verifyReCaptcha` API.
+
+   Retrieve the reCAPTCHA token via the Google API.
+
+</Steps>
+
+</Tasks>
+
+### Flow for forms covered by Adobe Commerce natively
+
+Use the following workflow for standard Adobe Commerce integrations
+
+<Steps>
+
+1.  Set the reCAPTCHA token in the X-ReCaptcha header.
+
+1.  Send the request to the backend.
+
+1.  Commerce validates the token. If it fails, Commerce returns an error.
+
+</Steps>
+
+### Flow for custom integrations
+
+Use one of the following workflows for custom integrations:
+
+- (Recommended) Send the token to Commerce and validate it on the server side. If validation passes, Commerce returns a normal response. Otherwise, it returns an error.
+
+- Validate the token on the client side. If validation passes, send the request.
+
+## reCAPTCHA API
+
+The reCAPTCHA package exposes the following APIs, allowing developers to integrate it with native Adobe Commerce functionality or set up custom integrations:
+
+- `setConfig`
+- `setEndpoint`
+- `initreCAPTCHA`
+- `verifyreCAPTCHA`
+- `enableLogger`
+
+### setConfig
+
+The `setConfig` method fetches and stores the reCAPTCHA configuration from the backend.
+It is essential to call this method before the `initreCAPTCHA` and `verifyreCAPTCHA` methods during app initialization, because other reCAPTCHA functionality will not be fully operational until the configurations are loaded and set up.
+
+#### Usage
+
+The `setConfig` method is intended to be called without parameters, in which case the backend configuration will be used.
+
+```javascript
+import * as reCAPTCHA from '@dropins/tools/reCAPTCHA.js';
+
+reCAPTCHA.setConfig();
+```
+
+However, you can pass a configuration object as a parameter to override the backend configurations.
+
+```javascript
+import * as reCAPTCHA from '@dropins/tools/reCAPTCHA.js';
+
+reCAPTCHA.setConfig({ is_enabled: true });
+```
+
+### setEndpoint
+
+The `setEndpoint` method sets the endpoint that the recaptcha module uses to fetch configurations. Its use is optional. If reCAPTCHA uses the same mesh endpoint as the rest of the application, you do not need to set it explicitly. It will be inherited from the fetch-graphql package.
```

Comment:
```suggestion
The `setEndpoint` method sets the endpoint that the reCAPTCHA module uses to fetch configurations. Its use is optional. If reCAPTCHA uses the same API Mesh endpoint as the rest of the application, you do not need to set it explicitly. It will be inherited from the `fetch-graphql` package.
```